src/components/FindPark.vue

```diff
@@ -13,10 +16,11 @@
     <input
       class="search-input"
       type="text"
+      aria-label="Type your search terms here"
```

Comment:
Really appreciate the aria-label as I have not focused on any of that yet with the new design and styling. Thank you!